src/pages/Proposal.vue

```diff
@@ -1,142 +1,137 @@
 <template>
-	<opt-in-error>
-		<a-row>
-			<a-col :span="12" :offset="6">
-				<div v-if="error" class="margin_bottom_sm">
-					<a-alert
-						message="Error"
-						:description="error"
-						type="error"
-						show-icon
-						closable
-						@close="error = ''"
+	<a-row>
```

Comment:
Why we are changing here for All dao?

Reply:
actually before we used to show an error on the screen if app was not opted in or dao id was not selected, but now instead of showing the error we disable the action(i.e for proposal, vote tokens, vote)

Reply:
Ok